src/twisted/web/test/test_agent.py

```diff
@@ -348,7 +348,8 @@ class StubEndpoint(object):
         def __init__(self, endpoint, testCase):
             self.endpoint = endpoint
             self.testCase = testCase
-            self.factory = _HTTP11ClientFactory(lambda p: None)
+            self.factory = _HTTP11ClientFactory(lambda p: None,
```

Comment:
This should be covered or ignored.  I suggest replacing `lambda p: None` with a named function that has a docstring but no body.